doc/manual/manual.tex

```diff
@@ -9921,6 +9921,42 @@ \subsubsection{Benchmarks for operator splitting}
+\begin{align}
+  \eta_{lin}(T,z) &= \exp(-\ln(\eta_T) * T + \ln(\eta_Z) * z)
+  \label{eq:tosi-benchmark-lin-visc}
+  \eta_{plast}(\dot\epsilon) &= \eta^* + \frac{\sigma_y}{\sqrt(\dot\epsilon:\dot\epsilon)}
+  \label{eq:tosi-benchmark-plast-visc}
+\end{align}
+where $\eta^*$ is the constant effective viscosity at high stresses and $\sigma_y$ the yield stress.
+
+\paragraph{Case 1: Temparature-dependent convection.}
+\label{sec:benchmark-tosi-case-1}
+
+The first benchmark considers a linear viscosity that only depends on temperature (Eq. \ref{eq:tosi-benchmark-lin-visc}, with $\gamma_Z=0$). When run to steady-state, this produces one convection cell with a high viscosity, stagnant lid insulating the fluid below (see Fig. 1 of \cite{T15}). In \cite{T15}, results of different codes are compared by looking at the average temperature, the Nusselt number at the top and bottom of the domain, the RMS velocity at the top boundary and in the whole domain and the maximum velocity at the surface. These quantities can be queried by using several of \aspect{} postprocessors, but the additional postprocessor in \url{benchmarks/tosi_et_al_2015_gcubed/tosi.cc} is needed to compute the average rate of work done against gravity, the average rate of viscous dissipation and the error between them. Differences between these diagnostic quantities of the 11 codes that participated in the benchmark effort are smaller than 5\% for their preferred mesh resolution.
+
+\paragraph{Case 2: Viscoplastic convection.}
+\label{sec:benchmark-tosi-case-2}
+Case 2 includes a strain rate dependent component in the viscosity, which is harmonically averaged with the linear component (see Eq. \ref{eq:tosi-benchmark-ave-visc} and the code snippet below).
+This rheology leads to mobile-lid convection, with the descending cold lid cooling the cell's interior (Fig. 2 of \cite{T15}).
+
+\begin{equation}
+  \eta_{T,\epsilon,z) = 2 \left(\frac{1}{\eta_{lin}}+\frac{1}{\eta_{plast}}\right)^{-1}  
+  \label{eq:tosi-benchmark-ave-visc}
+\end{equation}
+
+\lstinputlisting[language=prmfile]{cookbooks/benchmarks/tosi/tosi_benchmark_2.prm}
+
+Case 3 includes a depth-dependent component for the viscosity, but no strain rate-dependence (Eq. \ref{eq:tosi-benchmark-lin-visc}). Case 4 considers a full temperature-, depth- and strain rate-dependent viscosity, while in case 5 the yield stress is varied to investigate the transitions from mobile-lid to periodic to stagnant-lid convection regimes.
+
+
```

Comment:
I think it would be nice to have a closing paragraph here, rather than let the discussion just peter out. Maybe something like

"The input files referenced above implement these benchmarks. As mentioned before, the \aspect{} results are presented in \cite{T15} along with the results by a number of other codes solving the same problem. Fig.~\ref{XXX} shows one example of the kind of solution one gets when one solves these benchmarks numerically."

And then add one or two figures just so there is something to look at.